app/config/events.php

```diff
@@ -19,6 +19,18 @@
     'account.update.prefs' => [
         'description' => 'This event triggers when the account preferences are updated.',
     ],
+    'account.recovery.create' => [
+        'description' => 'This event triggers when the account recovery token is created.',
+    ],
+    'account.recovery.update' => [
+        'description' => 'This event triggers when the account recovery token is validated.',
```

Comment:
Is there a specific reason why you named this `.update` instead of `validate`? 
Just curious since i was confused when reading the name of the hook, but it was clear after i read the description 🤔

Reply:
same for `account.verification.update`. Does it makes sense to call it `account.verification.validated`?

Reply:
Because we're trying to stay consisted with REST standards. We're using POST request (create) to create token X, and we use PUT request (update) to verify it.

Reply:
I understand, makes sense :)